src/java.base/share/classes/sun/nio/fs/Util.java

```diff
@@ -38,7 +37,7 @@ class Util {
     private Util() { }
 
     private static final Charset jnuEncoding = Charset.forName(
-        GetPropertyAction.privilegedGetProperty("sun.jnu.encoding"));
+        System.getProperty("sun.jnu.encoding"));
```

Comment:
`StaticProperty.jnuEncoding()` can be used

Reply:
Fixed in commit 791663e.